api/base/filters.py

```diff
@@ -177,6 +177,8 @@ def convert_key(self, field_name, field):
         :param basestring field_name: text representation of the field name
         :param rest_framework.fields.Field field: Field instance
         """
+        if getattr(field, 'field', None):
```

Comment:
We probably shouldn't need this; is this to handle the `AllowMissing` field? If so, we might need to set `self.source` of `AllowMissing` fields to the inner `field's` source, though I'm not sure if this will have unintended side effects. It's worth a shot; if that doesn't work, then this can stay.